docker_entrypoint.sh

```diff
@@ -1,33 +1,55 @@
 #!/bin/bash
 
-set -e
+BITMONERO_DIR="/data/.bitmonero"
+MONERO_LOGS_DIR="$BITMONERO_DIR/logs"
+MONERO_WALLET_DIR="$BITMONERO_DIR/wallets"
+mkdir -p $MONERO_LOGS_DIR $MONERO_WALLET_DIR
+MONERO_LOG="$MONERO_LOGS_DIR/monerod.log"
+MONERO_WALLET_RPC_LOG="$MONERO_LOGS_DIR/monero-wallet-rpc.log"
+#monero.conf
+conf_template="/root/monero.conf.template"
+config_file="$BITMONERO_DIR/monero.conf"
+mv $conf_template $config_file
 
-BITMONERO_DIR=$(cat /root/monero.conf.template |grep "^data-dir="|sed "s/^data-dir=\([^ #]*\).*/\1/")
-cp /root/monero.conf.template $BITMONERO_DIR/
-new_conf_template="$BITMONERO_DIR/monero.conf.template"
-new_conf="$BITMONERO_DIR/monero.conf"
+#monero-wallet-rpc.conf
+wallet_rpc_conf_template="/root/monero-wallet-rpc.conf.template"
+wallet_rpc_conf="$BITMONERO_DIR/monero-wallet-rpc.conf"
+mv $wallet_rpc_conf_template $wallet_rpc_conf
 
-TOR_HOSTNAME=$(ip -4 route list match 0/0 | awk '{print $3}')
+MONERO_LAN_HOSTNAME="monerod.embassy"
+MONEROD_LOCAL_HOST="127.0.0.1"
+MONEROD_BIND_IP="0.0.0.0"
+TOR_SOCKS_PROXY_HOST=$(ip -4 route list match 0/0 | awk '{print $3}')
 TOR_PORT=9050
-STARTOS_PROXY_PORT=443
+STARTOS_REVPROXY_PORT=443
 MONERO_P2P_PORT=18080
 MONERO_RPC_PORT=18081
 MONERO_ZMQ_PORT=18082
-MONERO_RPC_PORT_HS=18083
+MONERO_ZMQ_PUBSUB_PORT=18083
+MONERO_P2P_PORT_LOCAL_BIND=18088
```

Comment:
```suggestion
MONERO_TOR_P2P_PORT=18084
```

Reply:
proposing the removal of a variable that is later used to replace a string in the config template means if someone enables the option that uses it, monerod is going to crash because of an invalid config.
Then you add a variable that's never used.  Why?
Obviously you have not tested your proposed change...

Reply:
This is the port being used for anonymous-inbound, is it not?

18088 should be reserved for monero-wallet-rpc (a few linen below this, you specift 28088 because youve already used 18088 for tor... should be 18088 for wallet-rpc and 18084 for tor)

18080 is default p2p
18081 is default node unrestricted rpc

18082 is default zmq RPC
18083 should be for zmq-pub

18084 for tor's anonymous-inbound
18085 for i2p's anonymous-inbound

18086 unused
18087 usused

18088 wallet-rpc
18089 node restricted rpc

and renaming the variable was to make it clear what it is being used for
`MONERO_P2P_PORT_LOCAL_BIND` really means tor's anonymous-inbound bind

https://github.com/kn0wmad/monerod-startos/pull/32#discussion_r1735398687

i didnt leave the old variable dangling. Changed here ^